src/server/appserver.cpp

```diff
@@ -96,6 +96,9 @@ static const char optionsC[] =
     "  --synthesis          : show the Synthesis window (if the application is running).\n";
 }
 
+static const QString showSynthesisMsg = "showSynthesis";
```

Comment:
What is the benefice of that? Since the arguments are passed to the programme as string, we will always needs to keep these keys as static string somewhere.
Same for the signal `QtSingleApplication::messageReceived`.
I feel like we would convert strings to enum without any gain in performance or readability. Do you have a suggestion?

Reply:
The benefits, if the list of such messages is liable to grow, are the following:
- Passing enums in function calls is faster than passing strings.
- Enums can be handled efficiently in switch statements, but not strings. Comparisons are faster than string comparisons.
- Enums are a natural and common way to define a family of several variables serving a common purpose.

If we opt for enums instead of strings, then we can and should turn these enums into strings at the time we want to display a string message. But before this moment, we benefit of the advantages of enums that I listed above.

As indicated in my comment, this becomes important only if the list of variables grow.

Reply:
I didn't realize that the API which allows us to pass messages from one server process to the other forces us to use `QString`. So what I suggested is less interesting than what I initially thought. The only advantage to use `enum`s in the specific context of this PR, would be to avoid a growing list of `static const QString` variables (those could be stored inside a map whose keys are `enums`). 

Up to you. Resolving this thread now.